packages/@react-facet/core/src/hooks/useFacetCallback.ts

```diff
@@ -137,7 +47,7 @@ export function useFacetCallback<M>(
         if (value === NO_VALUE) return NO_VALUE
       }
 
-      return callbackMemoized(...values)(...args)
+      return callbackMemoized(...(values as ExtractFacetValues<T>))(...(args as K))
```

Comment:
Note to other reviewers: the casting is here only because callbackMemoized has better types now. Before it accepted unknown[], that's why it didn't complain.
No regression here